api/src/main/java/io/grpc/Metadata.java

```diff
@@ -706,14 +815,36 @@ public String toString() {
      * @return a parsed instance of type T
      */
     abstract T parseBytes(byte[] serialized);
+
+    /**
+     * @return whether this key should be serialized to bytes lazily.
+     */
+    boolean serializeToStreams() {
+      return false;
+    }
+
+    /**
+     * Get this keys (implementation-specific) marshaller, or null if the
+     * marshaller is not of the given time.
```

Comment:
I don't understand `the given time` here, as  `marshaller` is a final field.

Reply:
Oops, that's a typo. time -> type.